src/js/components/githubIssues/selectIssueModal.tsx

```diff
@@ -228,24 +252,30 @@ const SelectIssueModal = ({
               </Trans>
             </p>
           </div>
-          <div
-            className="slds-grid
-              slds-grow
-              slds-shrink-none
-              slds-grid_align-end"
-          >
+          <div className="slds-grid slds-size_1-of-1 slds-medium-size_4-of-12">
             <ResyncIssuesButton
               projectId={projectId}
               isRefreshing={currentlyResyncing}
-              forceResync={Boolean(
-                issues &&
-                  attachedIssues &&
-                  !issues.length &&
-                  !attachedIssues.length,
-              )}
+              hasFetched={Boolean(issues && attachedIssues)}
+              noIssues={Boolean(!issues?.length && !attachedIssues?.length)}
             />
           </div>
         </div>
+        <div
+          className="slds-m-top_small
+            search-container
+            slds-size_1-of-1
+            slds-medium-size_8-of-12
+            slds-p-right_medium
+            slds-p-bottom_large"
```

Comment:
@dvdherron I probably didn't have clean code when I sent my gist, but maybe we should group the spacing classes together. 
```suggestion
          className="search-container
            slds-size_1-of-1
            slds-medium-size_8-of-12
            slds-m-top_small
            slds-p-right_medium
            slds-p-bottom_large"
```

I also wouldn't mind removing the top padding here and adding bottom padding to the Re-sync Issues button container, but that is really subjective.

Reply:
rearranged 👍🏽